packages/venia-concept/src/components/Navigation/__tests__/categoryTree.spec.js

```diff
@@ -0,0 +1,196 @@
+                        {
+                            id: 3,
+                            name: 'Child Category 2',
+                            position: 2,
+                            level: 1,
+                            url_key: 'child-2',
+                            url_path: 'test-category/child-2',
+                            product_count: 3,
+                            children_count: '1',
+                            path: '1/3'
+                        }
+                    ]
+                }
+            }
+        }
+    },
+    {
+        request: {
+            query: navigationMenuQuery,
+            variables: {
+                id: 2
+            }
+        },
+        result: {
+            data: {
+                category: {
+                    id: 2,
+                    name: 'Child Category 1',
+                    product_count: '5',
+                    path: '1/2',
+                    children: [
+                        {
+                            id: 4,
+                            name: 'Child Leaf 1',
+                            position: 1,
+                            level: 2,
+                            url_key: 'leaf-1',
+                            url_path: 'test-category/child-1/leaf-1',
+                            product_count: 4,
+                            children_count: '0',
+                            path: '1/2/4'
+                        },
+                        {
+                            id: 5,
+                            name: 'Child Leaf 2',
+                            position: 2,
+                            level: 2,
+                            url_key: 'leaf-2',
+                            url_path: 'test-category/child-1/leaf-2',
+                            product_count: 2,
+                            children_count: '0',
+                            path: '1/2/5'
+                        }
+                    ]
+                }
+            }
+        }
+    },
+    {
+        request: {
+            query: navigationMenuQuery,
+            variables: {
+                id: 3
+            }
+        },
+        result: {
+            data: {
+                category: {
+                    id: 3,
+                    name: 'Child Category 2',
+                    product_count: 3,
+                    path: '1/3',
+                    children: [
+                        {
+                            id: 6,
+                            name: 'Child Leaf 3',
+                            position: 1,
+                            level: 2,
+                            url_key: 'leaf-2',
+                            url_path: 'test-category/child-2/leaf-3',
+                            product_count: 6,
+                            children_count: '0',
+                            path: '1/3/6'
+                        }
+                    ]
+                }
+            }
+        }
+    }
+];
+
+const classes = {
+    root: 'a',
+    inactive: 'b'
```

Comment:
Now that we've merged #552, you don't need this particular hack.

Reply:
True. There are a lot of tests written this way for now.